osmnx/utils.py

```diff
@@ -205,10 +205,13 @@ def config(
         OSM "node" tags to add as graph node attributes, when present
     useful_tags_way : list
         OSM "way" tags to add as graph edge attributes, when present
-    session : requests.Session
-        session used by downloader for making HTTP requests to external APIs
-        override to provide proxies, client/server SSL certificates, or
-        basic authentication using username and password
+    session_config : dict
+        dictionary that supports overriding configuration for requests Session
+        to be used to during external API requests. currently supports auth, cert,
+        and verify as keys. auth is a tuple of (username, password). cert is the
+        path to a single file (containing the private key and certificate) or a tuple
+        of both files' paths. verify can be the path to a CA_BUNDLE file, or directory
+        of trusted CAs.
```

Comment:
It might be easier and more future-proof to just point the user toward the requests documentation on requests.Session.auth, cert, and path since we're just passing these values through to those requests.Session attributes.